Python/Password Checker/README.md

```diff
@@ -0,0 +1,20 @@
+# Password_Checker
+
+This self explanatory script, uses [PawnedPassword_API](https://pawnedpasswords.com).
+It checks how many times your password has been under hacaker radars.
+It's pretty cool.
+
+## Setup instructions and Guidelines
+
+This setup has only one requirement, `request`. Usually it is auto-downloaded with other packages.
+
+If that's not the case `pip  install request` will do the job.
```

Comment:
Add a `requirements.txt` file and ask the User to install packages from there, for the purpose.